_data/nav.yml

```diff
@@ -13,11 +13,11 @@
     - name: GTFS
       path: http://www.transitchicago.com/developers/gtfs.aspx
     - name: Health Atlas API
-      path: https://api.chicagohealthatlas.org/api-docs/index.html
+      path: https://rapidapi.com/blog/directory/chicago-police-clearpath/
```

Comment:
This is not the health atlas API.  Also this API documentation is from a third party, rapidapi.com.